scss/components/table.scss

```diff
@@ -176,9 +190,6 @@ $block: #{$fd-namespace}-table;
     &__fixed-col {
```

Comment:
same here - should this be `__fixed-cell`?

Reply:
same reply as above.